src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacEventPage.java

```diff
@@ -466,5 +470,11 @@ public void setAllowGroupReservations(final Boolean allowGroupReservations) {
     this.allowGroupReservations = allowGroupReservations;
   }
 
+  public Boolean getIsPrivateEvent() {
```

Comment:
When I suggested changing the name of `privateEvent` to `isPrivateEvent`, the getter was the main method I had in mind, because it's more readable to have something called `event.isPrivateEvent()` than `event.PrivateEvent()` or `event.getIsPrivateEvent()`. Normally the convention for boolean getters is to use `is` rather than `get`. So this should be `isPrivateEvent`. Some people like to leave the private value as `privateEvent` (rather than changing it to `isPrivateEvent`), and the setter to `setPrivateEvent` rather than `setIsPrivateEvent`. Do it as you like, but the getter should definitely be `isXYZ` for booleans.

Reply:
Ah, makes sense. Missed that. Should be simple enough to change at least...